### PR TITLE
Makes the share and back icons in the single speaker view easier to see

### PR DESCRIPTION
**Pull request by KnifesmithCode**
This samples a pixel from the top right corner of the speaker's image and uses that to check whether the icons should be black or white, then applies the color filter. Fixes issue #961

### CODE REVIEW

**Comment by KnifesmithCode** [comment]
I'm sorry, but I'm not 100% sure what's causing this error. I ran gradlew build on my own PC, and it worked fine.

**Comment by mananwason** [comment]
It's not able to find the drawable you have referenced. Please check. Thanks!

**Comment by the-dagger** [comment]
Probably the drawables were not pushed, happens a lot in android studio.
Please manually `git add` and then commit those drawables.
>
>

-- 
Best,
Harshit Dwivedi

**Comment by KnifesmithCode** [comment]
Okay, thanks. I added the drawables to the repository and pushed, and now it works

**Comment by the-dagger** [comment]
Looks good but there seems to be a merge conflict.
Please resolve it so that i can merge it.

File: android/app/src/main/java/org/fossasia/openevent/activities/SpeakerDetailsActivity.java [hunk 1]
```diff
@@ -224,9 +228,68 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+    protected void onPause() {
+        super.onPause();
+
+        Drawable shareDrawable = ContextCompat.getDrawable(getApplicationContext(), R.drawable.ic_share_white_24dp);
+        shareDrawable.clearColorFilter();
+
+        Drawable backDrawable = ContextCompat.getDrawable(getApplicationContext(), R.drawable.ic_arrow_back_white_24dp);
+        backDrawable.clearColorFilter();
+    }
 }
```
Comment (Abhi2424shek) [inline android/app/src/main/java/org/fossasia/openevent/activities/SpeakerDetailsActivity.java]: It would be better if you had a new line at EOF

**Comment by the-dagger** [comment]
Sadly there seems to be a merge conflict again.
Please resolve it.
I'll make it sure that there are no more merge conflicts for this PR.

**Comment by KnifesmithCode** [comment]
Okay, I'll see if I can sort everything out.

**Pull request merged**